webui/src/components/NewPoolsTable.js

```diff
@@ -123,43 +124,159 @@ const PoolsTable = ({ onPairClick }) => {
+          const poolAddressColumnIndex = columns.findIndex(
+            (col) => col === "address"
+          );
+
+          if (ownerColumnIndex >= 0) {
+            const owner = getRowValue(rowIndex, ownerColumnIndex);
+            const poolAddress = getRowValue(rowIndex, poolAddressColumnIndex);
+            await provider.send("eth_requestAccounts");
+            const signer = provider.getSigner();
+            const signerAddress = await signer.getAddress();
+
+            if (owner != signerAddress) {
+              alert(`Please provide correct owner address = ${owner}`);
+            } else {
+              const lssvmpair = contracts.pair(poolAddress, signer);
+              let tnx = await lssvmpair.changeDelta(
+                ethers.utils.parseEther(value)
+              );
+              await tnx.wait();
+            }
+          }
+        }}
       />
     );
   };
 
+  const ethBalanceCellRenderer = (rowIndex, columnIndex) => {
+    const value = ethers.utils.formatEther(getRowValue(rowIndex, columnIndex));
+    if (!value) return <Cell></Cell>;
+    return <EditableCell isFocused value={value} />;
+  };
+
+  const feeCellRenderer = (rowIndex, columnIndex) => {
+    const value = ethers.utils.formatEther(getRowValue(rowIndex, columnIndex));
+    const poolTypeColumnIndex = columns.findIndex((col) => col === "poolType");
+    const poolType = getRowValue(rowIndex, poolTypeColumnIndex);
+
+    if (!value) return <Cell></Cell>;
+    if (poolType == "2") {
+      return (
+        <EditableCell
+          isFocused
+          value={value}
+          onCancel={(value) => {
+            console.error(value);
+          }}
+          onChange={(value) => {
+            console.log(value);
+          }}
+          onConfirm={async (value) => {
+            // create transaction to update spot price
+            const ownerColumnIndex = columns.findIndex(
+              (col) => col === "owner"
+            );
+            const poolAddressColumnIndex = columns.findIndex(
+              (col) => col === "address"
+            );
+
+            if (ownerColumnIndex >= 0) {
+              const owner = getRowValue(rowIndex, ownerColumnIndex);
+              const poolAddress = getRowValue(rowIndex, poolAddressColumnIndex);
+              await provider.send("eth_requestAccounts");
+              const signer = provider.getSigner();
+              const signerAddress = await signer.getAddress();
+
+              if (owner != signerAddress) {
+                alert(`Please provide correct owner address = ${owner}`);
+              } else {
+                const lssvmpair = contracts.pair(poolAddress, signer);
+                let tnx = await lssvmpair.changeFee(
+                  ethers.utils.parseEther(value)
+                );
+                await tnx.wait();
+              }
+            }
+          }}
+        />
+      );
+    }
+  };
+
   return (
     <div style={{ height: 500 }}>
```

Comment:
@bisbist I think we should remove the height and add width value instead, so that the table doesn't extend full width. Can you add the following style to this div?
`width: 90%;
  margin: 20px auto;`